src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/redridge_mountains.cpp

```diff
@@ -35,68 +35,48 @@ enum
 {
     QUEST_MISSING_IN_ACTION = 219,
 
-    SPELL_MOCKING_BLOW      = 21008,
-    SPELL_SHIELD_BASH       = 11972,
-
-    SAY_CORPORAL_KEESHAN_1  = -1000561,
-    SAY_CORPORAL_KEESHAN_2  = -1000562,
-    SAY_CORPORAL_KEESHAN_3  = -1000563,
-    SAY_CORPORAL_KEESHAN_4  = -1000564,
-    SAY_CORPORAL_KEESHAN_5  = -1000565,
+    PATH_ID                 = 349,
```

Comment:
Use something non-generic pls. C++ has a really stupid way of mangling enums, and down the line this might be annoying. Smth like KEESHAN_PATH_ID would easily do the trick.